### PR TITLE
Add "make snapshot" target for building packages from develop branch

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
tendrl-bug-id: Tendrl/api#209

Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat ,  @mbukatov , @dahorak , please review

**Comment by TimothyAsir** [comment]
@anivargi Please review

File: Makefile [hunk 1]
```diff
@@ -40,3 +41,19 @@ rpm:
 		printf "\nThe tendrl-api RPMs are located at:\n\n"; \
 		printf "   $(DEPLOY)/latest\n\n\n\n"; \
 	fi
+
+srpm:
+	@echo "target: rpm"
```
Comment (ktdreyer) [inline Makefile]: bad copy & paste here? "srpm" is the target name

Comment (TimothyAsir) [inline Makefile]: Yes, thank you, I will update :)

File: Makefile [hunk 2]
```diff
@@ -40,3 +45,9 @@ rpm:
 		printf "\nThe tendrl-api RPMs are located at:\n\n"; \
 		printf "   $(DEPLOY)/latest\n\n\n\n"; \
 	fi
+
+update-release:
+	sed -i tendrl-api.spec \
+	  -e "/^Release:/cRelease: $(shell date +"%m_%d_%Y_%H_%M_%S")"
```
Comment (dahorak) [inline Makefile]: Please see my comment under the related issue related to the version format https://github.com/Tendrl/api/issues/209#issuecomment-314371093.

**Comment by TimothyAsir** [comment]
@ktdreyer , @mbukatov , @dahorak , Please review

**Comment by r0h4n** [comment]
Please send the PR on "master" branch

**Pull request closed without merging**